ush/HWP_tools.py

```diff
@@ -79,6 +101,18 @@ def copy_missing_restart(nwges_dir, hwp_non_avail_hours, hourly_hwpdir):
     return(restart_avail_hours, restart_nonavail_hours_test)
 
 def process_hwp(fcst_dates, hourly_hwpdir, cols, rows, intp_dir, rave_to_intp):
+    """Check the restart files.
+
+    Args:
+         fcst_dates: Forecast dates
+         hourly_hwpdir: ???
+         cols: number of columns
+         rows: number of rowa
+         intp_dir: directory
+         rave_to_intp: ???

```

Comment:
What do these params mean?

Reply:
fcst_dates is a list of forecasts (for production/ops/ebb=2, it is {current_day - 25 hours: current_day-1hours}, for ebb=1, it is current_day:current_day+24 hours)

hourly_hwpdir (see above)

cols/rows = hard coded dimensions for the RRFS_NA_3km and RRFS_CONUS_3km domains: cols, rows = (2700, 3950) if predef_grid == 'RRFS_NA_3km' else (1092, 1820)

intp_dir = is the working directory for smoke processing (${CYCLE_DIR}/process_smoke)

rave_to_intp = a string based on the grid name:  rave_to_intp = predef_grid+"_intp_"